get-started.md

```diff
@@ -0,0 +1,23 @@
+# How to get started in the Grafana Labs bug bounty program
+
+* **Grafana OSS** - we offer docker images that are synced with the `main` branch. This makes it easy to get started right away and be sure that you're looking for issues in an untouched instance. To spin up Grafana OSS in docker, run the following command:
+
+    `docker run -p 3000:3000 grafana/grafana-oss:main` and then visit http://localhost:3000 and login with `admin:admin`
```

Comment:
This was something I thought about, but simply copying a command is easier than reading a documentation page. I wonder how persistent these commands are. The intent of this document is to be hands-on, minimizing the time it takes to get started looking for vulnerabilities. 

Is there something specific i the text that you feel can become obsolete?

Reply:
Yeah - it's very hard to point at anything and say it will change (I would love for `admin:admin` to become outdated!). And just as equally, URLs absolutely become outdated too. Hence it wasn't a strong recommendation, just a suggestion - happy for you not to take it!